Makefile

```diff
@@ -89,6 +89,11 @@ fmt:
 .PHONY: lint
 lint:
 	$(LINT) run
+	prettier --check $(ALL_MD)
+
+.PHONY: lint-fix
+lint-fix:
```

Comment:
golangci-lint also has an auto-fix option. go ahead and add it here.

Reply:
Good call, I'll add that.